.github/workflows/mvn-build.yml

```diff
@@ -19,3 +19,5 @@ jobs:
           cache: 'maven'
       - name: Build with Maven
         run: mvn --update-snapshots -DtrimStackTrace=false  -Dsurefire.useFile=false -DskipITs verify
```

Comment:
How about removing `-DskipITs` to run all tests annotated with `@Tag("integration")` instead of making this one integration-test a unit-test by removing the `@Tag`?